Misc/NEWS.d/next/Windows/2021-04-08-00-36-37.bpo-34816.4Xe0id.rst

```diff
@@ -0,0 +1 @@
+`hasattr(ctypes.windll, 'nonexistant')` now returns `False` instead of raising `OSError`.

```

Comment:
Eh, rst is a bit more picky than markdown, so the docs are failing here because we should be using double ticks. I think I can fix it myself via a suggestion.

```suggestion
``hasattr(ctypes.windll, 'nonexistant')`` now returns `False` instead of raising exc:`OSError`.

```

Reply:
Thank you, @FFY00 !